sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -38,19 +38,12 @@
 
     public CouchConfig getCouchConfig(String db) {
 
+        //Android is always a specified couch via the AndroidTest runner
```

Comment:
isn't this comment misleading - doesn't SPECIFIED_COUCH have uses outside of android testing?